test/common/ops/releaseMounts.js

```diff
@@ -23,87 +23,85 @@ describe('shared.ops.releaseMounts', () => {
     user.balance = 1;
   });
 
-  it('returns an error when user balance is too low', done => {
+  it('returns an error when user balance is too low', async () => {
     user.balance = 0;
 
     try {
-      releaseMounts(user);
+      await releaseMounts(user);
     } catch (err) {
       expect(err).to.be.an.instanceof(NotAuthorized);
       expect(err.message).to.equal(i18n.t('notEnoughGems'));
-      done();
     }
   });
 
-  it('returns an error when user does not have all pets', done => {
+  it('returns an error when user does not have all pets', async () => {
     const mountsKeys = Object.keys(user.items.mounts);
     delete user.items.mounts[mountsKeys[0]];
 
     try {
-      releaseMounts(user);
+      await await releaseMounts(user);
```

Comment:
'nother double await. Might want to do a quick `grep` on this pattern, if it's not intentional